rust/src/wallet.rs

```diff
@@ -520,19 +521,20 @@ pub async fn open_channel(peer_info: PeerInfo, taker_amount: u64) -> Result<()>
 /// If the first channel is not usable, it might be because we've lost
 /// the connection with the 10101 maker, according to the
 /// `rust-lightning` logs.
-pub fn is_first_channel_usable() -> Result<bool> {
-    let channel_manager = {
-        let lightning = &get_wallet()?.lightning;
-
-        lightning.channel_manager.clone()
-    };
-
-    let is_usable = match channel_manager.list_channels().first() {
+pub fn is_first_channel_usable() -> bool {
```

Comment:
note: given that the channels might appear in any order, we will need to rework this when we decide to have more than one channel open (e.g. simultaneous cfd and lightning functionality)